### PR TITLE
Basic structure for index page. Folder structure for style sheets.

### PR DESCRIPTION
**Pull request by PhilBroderick**
Started the basic structure for the index page. Added a style sheet and the folder structure for assets. The navigation links aren't final. Just a placeholder.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,29 @@
...
+    <title>Code Coop</title>
+    <link rel="stylesheet" href="assets/css/index.css">
   </head>
   <body>
-      <h1>Hello World!</h1>
+    <header>
+        <h1>Code Coop</h1>
+    </header>
+      <nav>
+        <a href="about.html">About</a>
```
Comment (BarryGallagher) [inline index.html]: Add a title attribute https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#Adding_supporting_information_with_the_title_attribute

File: index.html [hunk 2]
```diff
@@ -2,9 +2,29 @@
...
+        <a href="contact.html">Contact Us</a>
```
Comment (BarryGallagher) [inline index.html]: Add a title attribute https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#Adding_supporting_information_with_the_title_attribute

File: index.html [hunk 3]
```diff
@@ -2,9 +2,29 @@
...
+        <a href="#">Other</a>
```
Comment (BarryGallagher) [inline index.html]: Add a title attribute https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#Adding_supporting_information_with_the_title_attribute

File: index.html (hunk 1 above)
Comment (BarryGallagher) [reply index.html]: This is a link to a file that doesn't exist yet (i.e. about.html). Would be good to add it now, even though it doesn't actually have content.

File: index.html (hunk 2 above)
Comment (BarryGallagher) [reply index.html]: This is a link to a file that doesn't exist yet (i.e. about.html). Would be good to add it now, even though it doesn't actually have content.

**Comment by PhilBroderick** [comment]
Thanks for the feedback Barry - going to make these changes now.

**Pull request closed without merging**